src/test/java/core/basesyntax/strategy/impl/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationHandlerTest {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+    private PurchaseOperationHandler purchaseOperationHandler = new PurchaseOperationHandler();
+
+    @BeforeEach
+    void setUp() {
+        Storage.SHOPSTORAGE.clear();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.SHOPSTORAGE.clear();

```

Comment:
I think you should clean the storage only 1 time, not 2

Reply:
agree here